src/Microsoft.OpenApi/Attributes/TrimmingAttributes.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#nullable enable
+
+namespace System.Diagnostics.CodeAnalysis
```

Comment:
The attributes that the trimmer uses to determine what should/shouldn't be trimmed aren't defined for the netstandard target. We're using the approach documented [here](https://devblogs.microsoft.com/dotnet/creating-aot-compatible-libraries/#approach-2-define-the-attributes-internally) to leverage them for our netstandard2.0 targeting packages.